.github/workflows/maintainer_merge_review.yml

```
@@ -28,7 +28,7 @@ jobs:
           type: 'stream'
           topic: 'maintainer merge'
           content: |
-            ${{ format('{0} requested a maintainer merge from review on PR [#{1}]({2}):', github.event.review.user.login, github.event.pull_request.number, github.event.pull_request.html_url ) }}
+            ${{ format('{0} requested a maintainer merge from review (action: {3}) on PR [#{1}]({2}):', github.event.review.user.login, github.event.pull_request.number, github.event.pull_request.html_url, github.event.action ) }}
```

```suggestion
            ${{ format('{0} requested a maintainer merge from reiew (action: {3}) on PR [#{1}]({2}):', github.event.review.user.login, github.event.pull_request.number, github.event.pull_request.html_url, github.event.action ) }}
```
I really like the idea of putting random typos!